plugins/openweather/commands/openweather_cmd.rb

```diff
@@ -0,0 +1,27 @@
+module AresMUSH
+  module Openweather
+    class OpenweatherCmd
+      include CommandHandler
+
+      def check_can_manage
+        return t('dispatcher.not_allowed') if !enactor.has_permission?("tinker")
```

Comment:
I don't think you want to limit this command to coders.

Reply:
This is an big oops on my part. Thank you for spotting it. Fixed.